src/PluginLoader.php

```diff
@@ -54,9 +53,7 @@ public function __construct($installedJson)
      */
     public function getInstances()
```

Comment:
Again, we should add `: array` here and to all other method declarations.

Reply:
done in b2ebae71102129721fe5765c6c1dd34af40e9b21